output/schema/validation-errors.json

```diff
@@ -236,6 +236,14 @@
       ],
       "response": []
     },
+    "indices.resolve_cluster": {
```

Comment:
I might be wrong, but I think these validation errors are related to a mismatch between what's in https://github.com/elastic/elasticsearch-specification/blob/main/specification/_json_spec/indices.resolve_cluster.json and in the ResolveClusterRequest.ts file.

I think the need for those *.json files will go away in the near future when these specifications move to the elasticsearch repo, but for now they're still used for validation purposes.